app/osparc.py

```diff
@@ -0,0 +1,103 @@
+from app.config import Config
+import json
+import osparc
+import tempfile
+from time import sleep
+
+
+def run_simulation(model_url, json_config):
+    temp_config_file = tempfile.NamedTemporaryFile(mode="w+")
```

Comment:
Use `with tempfile.NamedTemporaryFile() as temp:` context

Reply:
Done in https://github.com/agarny/sparc-api/commit/74676553fbb07339ea26653f8d2c0ad1508ef846.